File: openapi-templates/go/api_extra/authentication_expire_all.go
// Deprecated: Please use `AppPortalAccess` instead.
func (authentication *Authentication) DashboardAccess(
	ctx context.Context,
	appId string,
	o *AuthenticationDashboardAccessOptions,
) (*models.DashboardAccessOut, error) {
	pathMap := map[string]string{
		"app_id": appId,
	}
	headerMap := map[string]string{}
	var err error
	if o != nil {
		serializeParamToMap("idempotency-key", o.IdempotencyKey, headerMap, &err)
		if err != nil {
			return nil, err
		}
	}
	return executeRequest[any, models.DashboardAccessOut](
		ctx,
		authentication.client,
		"POST",
		"/api/v1/auth/dashboard-access/{app_id}",
		pathMap,
		nil,
		headerMap,
		nil,
	)
}

File: codegen-templates/ruby/api_extra/authentication_expire_all.rb
def dashboard_access(app_id, options = {})
  options = options.transform_keys(&:to_s)
  res = @client.execute_request(
    "POST",
    "/api/v1/auth/dashboard-access/#{app_id}",
    headers: {
      "idempotency-key" => options["idempotency-key"]
    }
  )
  DashboardAccessOut.deserialize(res)
end
